indico/web/client/js/jquery/index.js

```diff
@@ -23,6 +23,8 @@ import 'moment/locale/uk';
 import 'moment/locale/zh-cn';
 import 'moment/locale/es';
 import 'moment/locale/fr';
+import 'moment/locale/en-ca';
```

Comment:
No strong opinion, but I'd add it on top since so far we always added new locales on top ;)